package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "codechain-stakeholder-sdk",
```

Comment:
You shouldn't change this field because `-js` postfix is redundant when it comes to the NPM package.